src/editor/Collaboration.js

```diff
@@ -12,14 +12,23 @@ define(function (require, exports, module) {
             // immediately ask for camera access
             autoRequestMedia: false
         });
+        //To be moved to the bramble API.
+        var hash = window.location.hash.replace(/^#/, "");
+        var m = /&?collaboration=([^&]*)/.exec(hash);
```

Comment:
we want to try to minimize string manipulations in urls as much as possible.

I think I made a mistake in the code. Here's what it's supposed to be:
```js
var query = (new URL(window.location.href)).searchParams;
this.room = query.get("collaboration") || <random string>;
```